README.rst

```diff
@@ -147,8 +114,6 @@ The output should look something like::
       qthelp     to make HTML files and a qthelp project
       devhelp    to make HTML files and a Devhelp project
       epub       to make an epub
-      latex      to make LaTeX files, you can set PAPER=a4 or PAPER=letter
-      latexpdf   to make LaTeX files and run them through pdflatex
```

Comment:
These can be retained, particularly if we're retaining all other types like epub, devhelp and the rest.  This is a list of all possible types, and latex will remain a possible type (which we don't use, like all the rest here).